programming/sr/ruggeduinos/index.md

```diff
@@ -41,7 +41,7 @@ R.ruggeduinos["752303138333517171B1"].something...
 </div>
 
 
-[Setting pin modes](#pinmodes) {#pinmodes}
+[Setting pin modes](#pin-modes) {#pin-modes}
```

Comment:
Probably worth checking for anything outside this repo which might link here. Ideally urls shouldn't change.

Reply:
No exact matches on links to #pinmodes are indexed by Google, Bing, DuckDuckGo, or Yandex. Agree changing URLs aren't great but this feels like an improvement. Happy to switch back if others disagree though :)

Reply:
:shipit: